docs/experiments-core17.md

```diff
@@ -19,12 +19,31 @@ index segment into one.
 
 After indexing is done, you should be able to perform a retrieval as follows:
 
-```
+```bash
 sh target/appassembler/bin/SearchCollection \
   -topicreader Trec -index lucene-index.core.pos+docvectors -bm25 \
-  -topics src/main/resources/topics-and-qrels/topics.701-750.txt -output run.core.300-450.600-700.bm25.txt
+  -topics src/main/resources/topics-and-qrels/topics.core17.nist.txt -output run.core17.nist.bm25.txt
 ```
 
 For the retrieval model: specify `-bm25` to use BM25, `-ql` to use query likelihood, and add `-rm3` to invoke the RM3 
 relevance feedback model (requires docvectors index).
 
+
```

Comment:
Reformat this to follow the other collections? I.e., make the documentation look consistent.

Reply:
I observed that many of the experiment docs have this line with the same wording